WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -1,9 +1,47 @@
 import UIKit
+import Combine
+import OSLog
 
 class CardReaderSettingsViewController: UIViewController {
+    private var viewmodel: CardReaderSettingsViewModel
+    private var previousChildView: CardReaderSettingsActiveChildView = .none
+    private var subscriptions = Set<AnyCancellable>()
+
+    required init?(coder: NSCoder) {
+        self.viewmodel = CardReaderSettingsViewModel()
+        super.init(coder: coder)
+
+        os_log("In CRSVC init")
```

Comment:
Ah! The cool stuff! We use DDLog directly, if this log is for yourself, to trace that the VC is initialized while building its implementation, then maybe a print is enough?

Reply:
Yep - this was just for me to monitor lifecycle as I built out the foundation. Will remove.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3699/commits/46c1941a4b75b3595b156f57b130316575113c0e